frontend/src/static/js/v_summary.js

```diff
@@ -516,10 +534,30 @@ window.vSummary = {
               fileTypeColors[fileType] = selectedColors[i];
               i = (i + 1) % selectedColors.length;
             }
+            if (!this.fileTypes.includes(fileType)) {
+              this.fileTypes.push(fileType);
+            }
           });
         });
         this.contributionBarFileTypeColors = fileTypeColors;
       });
+
+      Object.keys(this.contributionBarFileTypeColors).forEach((fileType) => {
+        const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(this.contributionBarFileTypeColors[fileType]);
+        const red = parseInt(result[1], 16);
+        const green = parseInt(result[2], 16);
+        const blue = parseInt(result[2], 16);
+
```

Comment:
Just wish to understand. Why are both `green` and `blue` using `result[2]`?

Reply:
Fixed